app/routes/start-application.js

```diff
@@ -13,7 +17,9 @@ module.exports = [{
       const sbi = request.query?.sbi ?? agreement?.organisation?.sbi
       if (sbi) {
```

Comment:
I think there's edge cases not handled here:
- what if SBI is invalid (too short, not numbers, too long, etc is given, we have the SBI schema, just needs added)
- what if SBI is valid but does not correspond to a organisation record?

Just an observation because this is outside the scope of this ticket and change.

Reply:
I think both this scenario, and the one where there is nothing displayed as an error when POSTing are not covered in this ticket as it doesn't state what the behaviour should be, what error message to be displayed, etc.